src-tauri/src/main.rs

```diff
@@ -117,6 +118,12 @@ fn is_debug_mode() -> bool {
     return cfg!(debug_assertions);
 }
 
+#[tauri::command]
+/// Returns true if linux compatabile
+fn linux_checks() -> bool {
+    return app::linux_checks_librs();
```

Comment:
You can also add it to the huge list of imports [at the top of `main.rs`](https://github.com/GeckoEidechse/FlightCore/blob/89d6f0d9f31fbf5d073abd3db0ca18516bc7774e/src-tauri/src/main.rs#L12-L17).

Reply:
what line(s), i don't get what you're talking about?

Reply:
I meant instead of doing `app::linux_check_librs()` you can also import `linux_check_librs` and then change `app::linux_check_librs()` to `linux_check_librs()`.

Reply:
Ooooh 8 thought you mean there were a bunch of command definitions up there.